src/pages/Login/Login.js

```diff
@@ -1,7 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './Login.scss';
+import { useNavigate } from 'react-router-dom';
 
 const Login = () => {
-  return <div>Login</div>;
-};
+  const navigate = useNavigate();
+  const [inputValue, setInputValue] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setInputValue(prev => ({ ...prev, [name]: value }));
+  };
+  const id = inputValue.id;
+  const pw = inputValue.pw;
```

Comment:
```suggestion
  const {id, pw} = inputValue;

  const handleInput = e => {
    const { name, value } = e.target;
    setInputValue(prev => ({ ...prev, [name]: value }));
  };
```

해당 내용 구조분해할당을 이렇게 해 줄 수 있겠습니다!

Reply:
감사합니다, 해당사항 수정하겠습니다!

Reply:
수정완료하였습니다!